sktime/datasets/tsc_dataset_names.py

```diff
@@ -37,6 +37,7 @@
     "AllGestureWiimoteY",
     "AllGestureWiimoteZ",
     "ArrowHead",
+    "AsphaltObstacles",
```

Comment:
There are two other Asphalt datasets other than this, may want to remove this for now or add the other new datasets. Not a blocker though.

Reply:
I think we need a proper rethink about how to handle all this tbh!